src/utils/inputs.ts

```diff
@@ -7,6 +7,7 @@ export const enableCache = core.getInput("enable-cache") === "true";
 export const cacheSuffix = core.getInput("cache-suffix") || "";
 export const cacheLocalPath = getCacheLocalPath();
 export const cacheDependencyGlob = core.getInput("cache-dependency-glob");
+export const pruneCache = (core.getInput("prune-cache") || "true") === "true";
```

Comment:
```suggestion
export const pruneCache = core.getInput("prune-cache") === "true";
```

The default is always set